### ISSUE TITLE
statics files lost

### ISSUE THREAD

**Issue by manuKunam** [fdemmer/django-weasyprint#12]
I just tried to install the new django-weasyprint with pip. Before, I was using the 0.1 version. Now i have the 0.5 version.

So I changed PDFTemplateResponseMixin to the new WeasyTemplateResponseMixin.

My view (just an inherit view of a classic TemplateView) was working perfectly fine before, but
now all the statics files inserted in my templates with static tag don't work (css and images).  

Any ideas?

**Comment by fdemmer** [comment]
The `WEASYPRINT_BASEURL` setting is used to determine the URL where weasyprint will look for static files. If it is not set, `STATIC_URL` is used.

See https://github.com/fdemmer/django-weasyprint/blob/v0.5.0/django_weasyprint/views.py#L31

Previously the root of the request URL was used if `WEASYPRINT_BASEURL` was unset.

See https://github.com/fdemmer/django-weasyprint/blob/v0.1/django_weasyprint/views.py#L18

I reverted the change to use `STATIC_URL`. It seems wrong, when it contains eg "/static".

See https://github.com/fdemmer/django-weasyprint/blob/master/django_weasyprint/views.py#L33

Please try latest `master` and report back!

**Comment by pulse-mind** [comment]
Hello,
Same for me with 0.5.0
It works fine with master. 
Is it possible for you to publish a 0.5.1 ?

Thank you

**Comment by pulse-mind** [comment]
For information to others, To migrate from 0.1 to 0.5.0 ( + master)

I had to rename 
get_filename into get_pdf_filename in WeasyTemplateView class

Because I override the class, I had to change import : 
`from django_weasyprint.views import PDFTemplateView`
by 
`from django_weasyprint.views import WeasyTemplateView`

**Comment by SalahAdDin** [comment]
@pulse-mind Don't use these view, create your custom view, these view does not get any object.

**Comment by pulse-mind** [comment]
Sorry I was not clear I defined my view like this : 
`class PDFView(WeasyTemplateView ):`

**Comment by SalahAdDin** [comment]
No, better some like this: 
```python
# PDF file view
class RecipePDFPrintView(WeasyTemplateResponseMixin, DetailView):
    model = Recipe
    context_object_name = 'recipe'
    template_name = 'recipe/recipe_detail_pdf.html'  # TODO
    pdf_stylesheets = [
        settings.STATIC_ROOT + 'dist/css/pdf.css',
    ]
```

**Comment by pulse-mind** [comment]
Thank you. 
Is it possible for you to do a release please? The master branch is working fine better than the 0.5.0

**Comment by SalahAdDin** [comment]
What?

**Comment by pulse-mind** [comment]
If I do
`pip install django-weasyprint==0.5.0`
static files are not displayed
If I use the master they are displayed

**Comment by SalahAdDin** [comment]
@fdemmer Man!

**Comment by fdemmer** [comment]
published 0.5.1

please refrain from using the bugtracker for discussions and calling me out to do something. it is meant for _bug reporting_. i get notifications for all comments and will act, when my spare time permits.
